packages/raccoon-city-ui/src/components/Main/ChessGrid/ChessListView/ChessListView.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Paper from '@material-ui/core/Paper';
+import {ListContainer} from './ChessListView.styled';
+import {isActive} from '../ChessGrid.utils';
+
+export const ChessListView = ({listData, filters}) => (
+    <div>
+        <TableContainer component={Paper}>
```

Comment:
I would rather keep using the data table. Since they render flats in chunks. Because too many flats will produce perf problems